frontend/lib/pages/cross-site-terms-opt-in.tsx

```diff
@@ -28,6 +28,10 @@ export function hasLoggedInUserAgreedToTerms(s: AllSessionInfo): boolean {
 
     case "NORENT":
       return s.onboardingInfo.agreedToNorentTerms;
+
+    case "EVICTIONFREE":
+      // TODO: Need to figure out if we need separate terms for EvictionFree!
+      return s.onboardingInfo.agreedToJustfixTerms;
```

Comment:
We should file a github or clubhouse issue for this to make sure we don't forget about it.